### PR TITLE
docs/Updates to Porting guide and Template docs.

### PR DESCRIPTION
**Pull request by NotRobSheridan**
Updates to Porting guide and Template docs.
Edited screenshots to bring them in line with the style guide.
Rearranged ToC slightly.

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Updates to Porting guide and Template docs.
Edited screenshots to bring them in line with the style guide.
Rearranged ToC slightly.

Mostly groundwork to add further updates to both the Template and Porting guide pages. 

#### Tests
Local Improbadoc build
Staging: https://docs-staging.improbable.io/unreal/1.0/index

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
 
-    To do this: either in a terminal window or your file manager, navigate to the root of the `StarterTemplate` and copy all of the files and folders below to your `<ProjectRoot>`:  
+    To do this: either in a terminal window or your file manager, navigate to the root of the `StarterTemplate` repository and copy all of the files and directories below to your `<ProjectRoot>`:  
 
     ``` cpp
-    \TP_SpatialGDK\spatial-resources\spatial\
-    \TP_SpatialGDK\spatial-resources\LaunchClient.bat 
-    \TP_SpatialGDK\spatial-resources\LaunchServer.bat
-    \TP_SpatialGDK\spatial-resources\ProjectPaths.bat
+    \TP_SpatialGDK\spatial\
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: These need the `spatial-resources` folder added

Comment (m-samiec) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: There's also two new scripts in this directory - `DeployGame.bat` and `BuildProject.bat`. Is there a better way to detail them all?

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 2]
```diff
@@ -89,26 +107,31 @@ Before porting your project:
...
 
-1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-1. Right-click your `.uproject` file and select **Switch Unreal Engine version**.
+1. In **File Explorer**, navigate to `<ProjectRoot>\<GameRoot>`.
+1. Right-click your `<YourProject>.uproject` file and select **Switch Unreal Engine version**.
 1. Select the path to the Unreal Engine fork you cloned earlier.
-1. In File Explorer, right-click your `.uproject` file and select **Generate Visual Studio project files**.
-1. Open the generated solution file in Visual Studio and in the Development Editor build configuration, compile and run the project.
+1. In **File Explorer**, right-click `<YourProject>`.uproject and select **Generate Visual Studio Project files**.
+1. In the same directory, double-click `<YourProject>`.sln to open it with Visual Studio.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: This `sln` file is generated by the previous step. Is it worth referencing that fact?

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md [hunk 3]
```diff
@@ -126,70 +149,53 @@ It is necessary to modify your `GameInstance` class to work properly with the GD
...
+    ![spatial game instance reparent]({{assetRoot}}assets/screen-grabs/spatial-game-instance-reparent.png)<br/>_Image: The Blueprint class settings screen_<br/>
 
 ### 6. Generate schema and a snapshot
-You need to generate [schema]({{urlRoot}}/content/glossary#schema) and generate a [snapshot]({{urlRoot}}/content/glossary#snapshot) to get your game's deployment started. To do this:
+You need to generate [schema]({{urlRoot}}/content/schema) and generate a [snapshot]({{urlRoot}}/content/generating-a-snapshot) before you start your deployment. To do this:
 
-1. In the Unreal Editor, on the [GDK toolbar]({{urlRoot}}/content/toolbars), click the **Schema** button to run the [Schema Generator]({{urlRoot}}/content/glossary#schema-generation).
-1. On the same toolbar, click the **Snapshot** button which will generate a snapshot for the map currently open in the editor.
+1. In the Unreal Editor, on the [GDK toolbar]({{urlRoot}}/content/toolbars), select **Schema** to run the [Schema Generator]({{urlRoot}}/content/glossary#schema-generation).
+1. On the same toolbar, select **Snapshot*, which will generate a snapshot for the map currently open in the editor.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]:
```suggestion
1. On the same toolbar, select **Snapshot**, which will generate a snapshot for the map currently open in the editor.
```

Comment (NotRobSheridan) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: Good spot. Cheers!

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md (hunk 1 above)
Comment (NotRobSheridan) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: Does `spatial-resources` need to be added or removed? Bit confused. Just had a look at my template project and there isn't a `spatial-resources` directory, must've overlooked this. 
Unfortunately, I don't think there's a better way to list all of the files you need to copy over. 

I'll get the new .bat scripts added to the list.

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md (hunk 2 above)
Comment (NotRobSheridan) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]:
```suggestion
1. In **File Explorer**, right-click `<YourProject>`.uproject and select **Generate Visual Studio Project files**. This automatically generates a Visual Studio solution file for your project called `<YourProject.sln>`
1. In the same directory, double-click `<YourProject>`.sln to open it with Visual Studio.
```

Does this work?

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md (hunk 1 above)
Comment (m-samiec) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: Yes, these need the `spatial-resources` folder added back in.

File: SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md (hunk 2 above)
Comment (m-samiec) [reply SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md]: Looks good!

**Pull request merged**